extensions/NexusKitten/moremotion.js

```diff
@@ -65,6 +65,7 @@
               },
             },
             extensions: ["colours_motion"],
+            hideFromPalette: true,
```

Comment:
@GarboMuffin Oops? I'm pretty sure this block should be visible on the block palette.

Reply:
oops

Reply:
think i have a comment in there that's also wrong, i said old version was 0-360 when its -90-270